packages/main/src/themes/Dialog.css

```diff
@@ -1,24 +1,27 @@
 :host {
 	min-width: 20rem;
 	min-height: 6rem;
+	max-height: 94%;
+	max-width: 90%;
+	flex-direction: column;
 	box-shadow: var(--sapContent_Shadow3);
 }
 
 :host([stretch]) {
 	width: 90%;
-	height: 90%;
+	height: 94%;
+}
+
+:host([on-phone]) {
+	max-height: 100%;
+	max-width: 100%;
```

Comment:
should be :host([stretch][on-phone]) { max-height: 100%; max-width: 100%; } (like it was)
because it's possible to have unstretched dialog on phones